network-api/networkapi/templates/pages/article_page.html

```diff
@@ -54,7 +54,11 @@
 
     <div class="cms article-page-content article-blocks">
         {% for block in page.body %}
-            {% include_block block with parent_page=page page_type="article" %}
+            {% if block.block_type == "accordion" %}
+                {% include_block block with parent_page=page page_type="article" width="4/5" %}
```

Comment:
Thanks for pointing this out! I did a find-all search in the html files in the codebase. Looks like `article_page.html` isn't the only file that passes `parent_page` and `page_type` to blocks. I think I'm gonna leave the PR as is for now for consistency purpose. Will file a ticket to do a proper audit.

<img width="485" alt="image" src="https://user-images.githubusercontent.com/2896608/228941071-bdbad078-d165-4db2-99cc-f239bb25b47f.png">


<img width="482" alt="image" src="https://user-images.githubusercontent.com/2896608/228940988-af87b57b-97f8-401e-ae79-f995c6b32892.png">

Reply:
That's fine. 

However, I would like to point out that this exact pattern is somewhat of an anti-pattern to me, because it leads conditionals inside the fragments that check for the environment. These kinds of conditionals make reuse of fragments pretty hard, because you might want to use the same fragment in a different context in which that check makes no sense. It would be better to control the fragment through arguments that say something about their effect.